Tests/Particles/AssignDensity/main.cpp

```diff
@@ -68,7 +65,7 @@ void test_assign_density(TestParams& parms)
   int iseed = 451;
   Real mass = 10.0;
 
-  MyParticleContainer::ParticleInitData pdata = {mass, AMREX_D_DECL(1.0, 2.0, 3.0)};
+  MyParticleContainer::ParticleInitData pdata = {mass, AMREX_D_DECL(1.0, 2.0, 3.0), {}, {}, {}};
```

Comment:
```suggestion
  MyParticleContainer::ParticleInitData pdata = {{mass, AMREX_D_DECL(1.0, 2.0, 3.0)}, {}, {}, {}};
```
Thanks for fixing these warnings. With clang, it also complains unless we insert these extra brackets around the real struct data part.

Reply:
Gotta love compiler bracket logic. 🤦